frontend/src/metabase/query_builder/components/expressions/analytics.ts

```diff
@@ -0,0 +1,10 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+import * as Lib from "metabase-lib";
+
+export const trackColumnCombineViaShortcut = (query: Lib.Query) => {
+  trackSchemaEvent("question", "1-0-4", {
```

Comment:
`question_id` is required and cannot be omitted unfortunately. You would also have to default to `0` for unsaved questions.